### ISSUE TITLE
Scrolling not working

### ISSUE THREAD

**Issue by angwandi** [flutter-form-builder-ecosystem/form_builder_image_picker#48]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package/Plugin version

4.0.0

### Flutter doctor

[√] Flutter (Channel stable, 3.10.0, on Microsoft Windows [Version 10.0.22000.1696], locale en-GB)
    • Flutter version 3.10.0 on channel stable at C:\Users\demad\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 84a1e904f4 (7 days ago), 2023-05-09 07:41:44 -0700
    • Engine revision d44b5a94c9
    • Dart version 3.0.0
    • DevTools version 2.23.1

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 33.0.2)
    • Android SDK at C:\Users\demad\AppData\Local\Android\sdk
    • Platform android-33, build-tools 33.0.2
    • Java binary at:
      C:\Users\demad\AppData\Local\JetBrains\Toolbox\apps\AndroidStudio\ch-0\222.4459.24.2221.9971841\jbr\bin\java
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-b2043.56-9586694)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.0.5)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.0.32112.339
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2021.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-8887301)

[√] Android Studio (version 2022.2)
    • Android Studio at C:\Users\demad\AppData\Local\JetBrains\Toolbox\apps\AndroidStudio\ch-0\222.4459.24.2221.9971841
    • Flutter plugin version 73.0.2
    • Dart plugin version 222.4582
    • Java version OpenJDK Runtime Environment (build 17.0.6+0-b2043.56-9586694)

[√] IntelliJ IDEA Ultimate Edition (version 2023.1)
    • IntelliJ at C:\Users\demad\AppData\Local\JetBrains\Toolbox\apps\IDEA-U\ch-0\231.8770.65
    • Flutter plugin version 73.1.1
    • Dart plugin version 231.9065

[√] VS Code (version 1.78.1)
    • VS Code at C:\Users\demad\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.46.0

[√] Connected device (4 available)
    • moto g 6 plus (mobile) • ZY32389KCQ • android-arm64  • Android 9 (API 28)
    • Windows (desktop)      • windows    • windows-x64    • Microsoft Windows [Version 10.0.22000.1696]
    • Chrome (web)           • chrome     • web-javascript • Google Chrome 113.0.5672.93
    • Edge (web)             • edge       • web-javascript • Microsoft Edge 112.0.1722.58

[√] Network resources
    • All expected network resources are available.

• No issues found!

### Minimal code example

<details>
  <summary>Code sample</summary>

```dart

```

</details>

### Current Behavior

When adding multiple images, the scrolling is not working as intended. It locks the scroll to the last 2 images thus not being able to scroll and see the other images added. 

### Expected Behavior

Horizontal smooth scroll

### Steps To Reproduce

Not sure

**Comment by deandreamatias** [comment]
Duplicated of https://github.com/flutter-form-builder-ecosystem/form_builder_image_picker/issues/6 ?

**Comment by SherpaMiguel** [comment]
> Duplicated of [#6](https://github.com/flutter-form-builder-ecosystem/form_builder_image_picker/issues/6) ?

Not really. This problem only occurs on web desktop for me. Click event does not work to scroll the ListView. Tap event works as expected with same screen dimensions.

**Comment by SherpaMiguel** [comment]
I don't know if it is the same bug, but I resolved mine wrapping ListView.builder like this:

```
import 'package:flutter/gestures.dart';

ScrollConfiguration(
 behavior: ScrollConfiguration.of(
   state.context,
 ).copyWith(dragDevices: {PointerDeviceKind.touch, PointerDeviceKind.mouse}),
 child: ListView.builder(
```
